Core/include/Acts/Visualization/PlyVisualization3D.hpp

```diff
@@ -60,6 +60,10 @@ class PlyVisualization3D : public IVisualization3D {
   /// @copydoc Acts::IVisualization3D::clear()
   void clear() final;
 
+  void object(const std::string& /*name*/) final {
```

Comment:
Should this throw a `std::runtime_error`?

Reply:
It could be, but I was thinking that since PLY still produces a valid output file even when objects are being used, it would make sense to have this be silent.

Reply:
We don't have a logger in the class, right?

Reply:
Currently not. We could add it, but outside of this so far there hasn't been much need for output messages.